README.md

```diff
@@ -188,6 +191,7 @@ API | Description | Auth | HTTPS | Link |
 | Genderize.io | Determines a gender from a first name | No | Yes | [Go!](https://genderize.io) |
 | Github - User Data | Pull public information for a user's github | No | Yes | [Go!](https://api.github.com/users/hackeryou) |
 | Gitter | Chat for GitHub | `OAuth` | Yes | [Go!](https://github.com/gitterHQ/docs) |
+| Google Speech | Converts audio to text | `OAuth` | Yes | [Go!](https://cloud.google.com/speech/) |
```

Comment:
GCP and AWS services are a tricky situation - both offers a lot of free tier APIs, but they come with a million strings attached. I'd like to hold off on adding GCP/AWS/Azure services for now, since those fall into a category unto themselves - I think they offer a lot of opportunity, but they are vastly different than most services on the list currently.

Reply:
Gotcha 👍 

Removing it.